_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,131 @@
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/TwoOptionWithLogo.png" alt="Multi-option sign-in window with with some customization" width="350">
+
+### Basic authentication settings
+
+The settings below are used to customize the basic username and password sign-in button.
+
+Setting | Description
+:--- | :--- |:--- |:--- |
+`opensearch_security.ui.basicauth.login.buttonname` |  Display name for the login button. "Log in" by default.
+`opensearch_security.ui.basicauth.login.brandimage` |  Login button logo. Supported file types are SVG, PNG, and GIF.
+`opensearch_security.ui.basicauth.login.showbrandimage` |  Determines whether a logo for the login button is displayed or not. Default is `true`. 
+
+### OpenID Connect authentication settings
+
+These settings allow you to customize the sign-in button associated with OpenID Connect authentication. For the essential settings required for single sign-in using OpenID Connect, see [OpenSearch Dashboards single sign-on]({{site.url}}{{site.baseurl}}/security-plugin/configuration/openid-connect/#opensearch-dashboards-single-sign-on).
+
+Setting | Description
+:--- | :--- |:--- |:--- |
+`opensearch_security.ui.openid.login.buttonname` |  Display name for the login button. "Log in with single sign-on" by default.
+`opensearch_security.ui.openid.login.brandimage` |  Login button logo. Supported file types are SVG, PNG, and GIF.
+`opensearch_security.ui.openid.login.showbrandimage` |  Determines whether a logo for the login button is displayed or not. Default is `false`.
+
+### SAML authentication settings
+
+These settings allow you to customize the sign-in button associated with SAML authentication. For the essential settings required for single sign-in using SAML, see [OpenSearch Dashboards configuration]({{site.url}}{{site.baseurl}}/security-plugin/configuration/saml/#opensearch-dashboards-configuration).
+
+Setting | Description
+:--- | :--- |:--- |:--- |
+`opensearch_security.ui.saml.login.buttonname` |  Display name for the login button. "Log in with single sign-on" by default.
+`opensearch_security.ui.saml.login.brandimage` |  Login button logo. Supported file types are SVG, PNG, and GIF.
+`opensearch_security.ui.saml.login.showbrandimage` |  Determines whether a logo for the login button is displayed or not. Default is `false`.
+
+## Sample setup
+The following example shows basic settings in the `opensearch_dashboards.yml` file  when configured for two options at sign-in.
+
+```yml
+server.host: 0.0.0.0
+server.port: 5601
+opensearch.hosts: ["https://localhost:9200"]
+opensearch.ssl.verificationMode: none
+opensearch.username: <preferred username>
+opensearch.password: <preferred password>
+opensearch.requestHeadersAllowlist: ["securitytenant","Authorization"]
+opensearch_security.multitenancy.enabled: true
+opensearch_security.multitenancy.tenants.preferred: ["Private", "Global"]
+opensearch_security.readonly_mode.roles: ["<role_for_read_only>"]
```

Comment:
Line 98 - 107: is optional

Reply:
Added a hash comment to clarify what the first 10 settings are for.